Dockerfile

```diff
@@ -30,6 +34,9 @@ RUN sed -i 's/initLimit=10/initLimit=100/' /opt/zookeeper/conf/zoo.cfg
 # Whitelist this commands
 RUN echo "4lw.commands.whitelist=stat, ruok, conf, isro" >> /opt/zookeeper/conf/zoo.cfg
 
+# Fix Unreasonable length issue - https://www.pivotaltracker.com/story/show/185950085
```

Comment:
Can we add the link to the apache issue comment here as well please?

Reply:
Yup done